front/app/containers/Admin/settings/messages.ts

```diff
@@ -28,6 +28,10 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SettingsPage.tabBranding',
     defaultMessage: 'Branding',
   },
+  tabStatuses: {
+    id: 'app.containers.AdminPage.SettingsPage.tabStatuses',
+    defaultMessage: 'Statuses',
```

Comment:
Maybe now it's not immediately clear what kind of statuses they are.
![image](https://github.com/CitizenLabDotCo/citizenlab/assets/91883434/2a889538-c307-4826-ab9d-701e869df665)

Maybe "Input statuses" could be better.